src/NCalc/NCalc.csproj

```diff
@@ -1,51 +1,73 @@
-
-  <!-- TRACE symbol seems to be always defined by default, let's undefine it for the Release builds -->
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants.Replace("TRACE", "NOTRACE"))</DefineConstants>
-  </PropertyGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' != 'net35' ">
-    <Compile Remove="Properties\ExcludeFromCodeCoverageAttribute35.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Antlr4.Runtime.Standard" Version="4.13.1" />
-  </ItemGroup>
-
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net35' ">
-    <DefineConstants>$(DefineConstants);</DefineConstants>
-    <FrameworkPathOverride>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <None Include="Resources\NCalc.png" Pack="true" PackagePath="" />
-    <None Include="..\..\README.md" Pack="true" PackagePath="" />
-  </ItemGroup>
-
-  <PropertyGroup Condition="'$(Configuration)'=='Release'">
-    <Deterministic>true</Deterministic>
-    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-  </PropertyGroup>
+	<!-- Common Property Groups -->
+
+	<PropertyGroup>
+		<TargetFrameworks>net461;netstandard2.0;net6.0;net8.0</TargetFrameworks>
+		<!-- Other common properties -->
+		<LangVersion>12</LangVersion>
+		<Authors>Sebastien Ros and contributors</Authors>
+		<Copyright>Sebastien Ros and contributors</Copyright>
+		<Company>https://github.com/ncalc</Company>
+		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+		<DefineConstants>$(DefineConstants.Replace("TRACE", "NOTRACE"))</DefineConstants>
+		<PackageProjectUrl>https://github.com/ncalc/ncalc</PackageProjectUrl>
+		<RepositoryUrl>https://github.com/ncalc/ncalc.git</RepositoryUrl>
+		<PackageTags>ncalc; sync; core; expressions;</PackageTags>
+		<PackageIcon>NCalc.png</PackageIcon>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+	</PropertyGroup>
+
+	<!-- Unsigned Assembly Configuration -->
+	<PropertyGroup Condition=" '$(Configuration)' == 'Debug' Or '$(Configuration)' == 'Release' ">
+		<AssemblyName>NCalc</AssemblyName>
+		<PackageId>NCalcSync</PackageId>
+		<Description>NCalc is a mathematical expressions evaluator for .NET
+
+NCalc can parse any expression and evaluate the result, including static or dynamic parameters and custom functions.
+
+This package contains the unsigned assembly 'NCalc'. For the stronly signed assembly 'NCalc.signed', see the NCalcSync.signed package.
+		</Description>
+	</PropertyGroup>
+
+	<!-- Strongly Signed Assembly Configuration -->
+	<PropertyGroup Condition=" '$(Configuration)' == 'SignedRelease' ">
+		<AssemblyName>NCalc.signed</AssemblyName>

```

Comment:
Is it necessary to change the assembly name?

Reply:
Good point. Users will decide for the signed or the unsigned package, but never for both. So a collision of assembly names is impossible in practice.
The PR is updated, so the assembly name is NCalc.dll for both unsigned and signed versions.